### ISSUE TITLE
Why trained models I re-produced had no effect in OpenPose demo

### ISSUE THREAD

**Issue by Omnivore-znc** [ZheC/Realtime_Multi-Person_Pose_Estimation#122]

**Comment by Omnivore-znc** [comment]
I've trained for more than one week and got many snapshot models， but when I used these models in OpenPose VC++project，the first 19 channels of output data are abnormal（specifically, no heatmap feature data）but the other channels  seem well (specifically, there exist PAF feature output)。I know there some differences between the .deploy file I re-produced with that you provide in Openpose, but whatever modfification I've done, it still unsolved. Can you help me in detail or give me some tips? 




[pose_deploy(re-produced).txt](https://github.com/ZheC/Realtime_Multi-Person_Pose_Estimation/files/1526526/pose_deploy.re-produced.txt)
[pose_deploy_linevec(provided in openpose demo).txt](https://github.com/ZheC/Realtime_Multi-Person_Pose_Estimation/files/1526528/pose_deploy_linevec.provided.in.openpose.demo.txt)
[pose_train_test(re-produced).txt](https://github.com/ZheC/Realtime_Multi-Person_Pose_Estimation/files/1526529/pose_train_test.re-produced.txt)

**Comment by YongboYu** [comment]
@Omnivore-znc Hi, I have trained this model by 4 GPUs, and I have the same paper as you. The caffemodel I trained did not work int the Openposehttps://github.com/CMU-Perceptual-Computing-Lab/openpose/tree/master/models/pose/coco. Did you deal with. If you got it ,would you mind sharing it! Thanks

**Comment by Omnivore-znc** [comment]
@YongboYu Yes, I‘ve done it. In fact, no problems actually exist in the train code. The problem I came with above appeared maybe due to the trainning parameters falled into local optimum in the final stage.

**Comment by Omnivore-znc** [comment]
@Ai-is-light I'm sorry. That log files might have been deleted for more than one month
